index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
[参考記事](https://style.potepan.com/articles/20050.html)
```suggestion
<html lang="ja">
```